tests/s2_inference/test_encoding.py

```diff
@@ -149,3 +153,50 @@ def test_model_un_normalization(self):
 
                 assert abs(max_output_norm - 1) > eps, f"{name}, {sentence}"
                 assert abs(min_output_norm - 1) > eps, f"{name}, {sentence}"
+
+    def test_open_clip_vectorize(self):
+
+        names = ['open_clip/RN50/openai', 'open_clip/RN101/yfcc15m', 'open_clip/RN101-quickgelu/openai',
```

Comment:
lets use two models here to avoid long testing, maybe `'open_clip/ViT-B-32/laion400m_e32'` and `'open_clip/RN50/openai'`

Reply:
![image](https://user-images.githubusercontent.com/49334982/194797040-aec51a64-a0b7-4821-9c3e-37c54e456a94.png)

resolved. testing time reduced from 5 mins to 3 mins